nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2743,6 +2743,64 @@ nifi.content.repository.encryption.key.id=Key1
 nifi.content.repository.encryption.key=0123456789ABCDEFFEDCBA98765432100123456789ABCDEFFEDCBA9876543210
 ....
 
+
+=== HDFS Content Repository Properties
+
```

Comment:
What are the security considerations of this new capability?  Does it support kerberos?  Does it support TDE/etc..?

Reply:
It is using the standard HDFS Hadoop Client API, so yes it would. I actually haven't it before, but it appears to be configured through the Hadoop core-site.xml which is the config file the repository (and other HDFS processors) requires.